src/Symbols/include/Symbols/SymbolLoadingOutcome.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SYMBOLS_SYMBOL_LOADING_OUTCOME_H_
+#define SYMBOLS_SYMBOL_LOADING_OUTCOME_H_
+
+#include <filesystem>
+#include <variant>
+
+#include "OrbitBase/CanceledOr.h"
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/NotFoundOr.h"
+#include "OrbitBase/Result.h"
+
+namespace orbit_symbols {
+
+struct SuccessOutcome {
```

Comment:
When reading this out of context, it might be a bit confusing. What about `SymbolLoadingResult` `SymbolLoadingDetails` or something similar?

Reply:
My initial thinking was this being clear because of the namespace, aka `orbit_symbols::SuccessOutcome`. I renamed it now to `orbit_symbols::SymbolLoadingSuccessResult`